data/json/requirements/toolsets.json

```diff
@@ -1,5 +1,11 @@
 [
   {
+    "id": "forging_standard",
+    "type": "requirement",
+    "//": "Forging of steel items (per steel chunk)",
+    "tools": [ [ [ "forge", 20 ], [ "char_forge", 20 ], [ "oxy_torch", 20 ] ] ]
```

Comment:
Second table. The value I think was derived from @Rivet-the-Zombie's later comment. We could temporarily scale oxy_torch down to 5 charges as I agree whilst probably correct IRL does upset balance somewhat.